project3/styles.css

```diff
@@ -0,0 +1,218 @@
+:root{
+  --color-red:
+}
+*, 
+*::before,
+*::after{
+  box-sizing: border-box;
+}
+
+body{
+  background-color: #163346;
```

Comment:
Remember to also include base rules for `font-family` and text `color` on your `body` element.